Fix GitHub publish workflow

GitHub publish workflow currently fails and returns a 401. Running `./gradlew publish` locally is currently required as a workaround for this.

#### Environment details

GitHub actions

#### Steps to reproduce

  1. Merge a PR that is supposed to trigger a build
  2. Notice that the publish workflow fails with a 401

#### Code example

```python
# example
```

#### Stack trace

```
Unzipping /home/runner/.gradle/wrapper/dists/gradle-6.1.1-all/6i0vmch5if35vdjupkkj651gut/gradle-6.1.1-all.zip to /home/runner/.gradle/wrapper/dists/gradle-6.1.1-all/6i0vmch5if35vdjupkkj651gut
Set executable permissions for: /home/runner/.gradle/wrapper/dists/gradle-6.1.1-all/6i0vmch5if35vdjupkkj651gut/gradle-6.1.1/bin/gradle

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':publishMapsJavaUtilsPublicationToMavenRepository'.
> Failed to publish publication 'MapsJavaUtils' to repository 'maven'
   > Could not PUT 'https://oss.sonatype.org/service/local/staging/deploy/maven2/com/google/maps/google-maps-services/0.15.0/google-maps-services-0.15.0.jar'. Received status code 401 from server: Unauthorized

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.
```

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -21,6 +21,7 @@ on:
       - '*'
   repository_dispatch:
     types: [publish]
+  workflow_dispatch:
 
 jobs:
   publish:
@@ -46,5 +47,5 @@ jobs:
         GPG_KEY_ARMOR: "${{ secrets.SYNCED_GPG_KEY_ARMOR }}"
         GPG_KEY_ID: ${{ secrets.SYNCED_GPG_KEY_ID }}
         GPG_PASSWORD: ${{ secrets.SYNCED_GPG_KEY_PASSWORD }}
-        SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
-        SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+        SONATYPE_PASSWORD: ${{ secrets.SYNCED_SONATYPE_PASSWORD }}
+        SONATYPE_USERNAME: ${{ secrets.SYNCED_SONATYPE_USERNAME }}

